### ISSUE TITLE
Possibly drop support for all but main model organisms + H.s.

### ISSUE THREAD

**Issue by ppavlidis** [PavlidisLab/gotrack#26]
Primary example is on the term tracker Gene Breakdown.

Organisms like dog, cow, pig, chicken are of lower use. Since most of these annotations are computational, and the number of people using them is small, I'm not sure it adds enough value. 

It should be a server configuration to choose which organisms to work with so we can re-enable them if we feel the need.

**Comment by ppavlidis** [comment]
Just to clarify, there's probably no major downside to keeping these orgs in our database and allowing the API to support them. Just about the web interface.

**Comment by JacobsonMT** [comment]
Similar server configuration already exists, the following will only load the organisms in a comma separated list:

gotrack.speciesRestriction=fly,yeast,zebrafish,dicty,arabidopsis,human,worm,mouse,rat

(Are these the organisms you'd like enabled on production?)

What is currently not supported is differentiating organisms loaded from organisms displayed thus allowing a situation where we display a subset of organisms in the UI but allow all in the API.

**Comment by ppavlidis** [comment]
Yes, that list is okay, dropping pig, dog and chicken is fine. Having a different setup for the API is not important.
